CMakeLists.txt

```diff
@@ -54,8 +54,8 @@ if(MSVC)
 	# Because our Exceptions derive from std::runtime_error we can safely disable this warning
 	set(PLATFORM_COMPILE_FLAGS "${PLATFORM_COMPILE_FLAGS} /wd4275")
 else()
-	#TODO: Fix warnings before enabling the flag.
-	#set(PLATFORM_COMPILE_FLAGS "${PLATFORM_COMPILE_FLAGS} -Wall")
+    # TODO: OCIO being under active development, unused functions are fine for now.
```

Comment:
[minor] Noting that this line uses spaces for indentation, where others are using tabs.

Reply:
Even if the file is using tabulations, the coding style requires to not use them. So, I will fix that for short-term i.e. put a tabulation instead of spaces.